libarchive/archive_read_disk_entry_from_file.c

```diff
@@ -727,6 +728,8 @@ setup_xattrs(struct archive_read_disk *a,
 
 	if (list_size == -1 && errno == EOPNOTSUPP)
 		return (ARCHIVE_OK);
+	if (list_size == -1 && errno == EPERM)
+		return (ARCHIVE_OK);
```

Comment:
bsdtar's test_option_acls changes behavior, please investigate and fix test and/or behavior

Reply:
Accessing the system filesystem extended namespace is a privileged operation. The reason for this conditional is to ignore permission denied when an unprivileged user attempts to access the system namespace.

Reply:
I am just writing this to check why does one of the bsdtar tests behave differently (fail) after this change on FreeBSD.